packages/builder/src/helpers/duplicate.ts

```diff
@@ -76,13 +76,15 @@ export const getSequentialName = <T extends any>(
   {
     getName,
     numberFirstItem,
+    separator = "",
```

Comment:
The way to handle a space is just to include a space in the prefix - so just add a space to the end of `New ${definition.friendlyName || definition.name}` down below and it has the same effect without needing a separator param 👌 This util is written to trim the prefix and handle it properly when looking for existing names.

Reply:
This was my initial approach, but I felt it was confusing. I set that the prefix (aka the name I actually want) with the whitespace, but the result is without the whitespace if there is a count. As a user, I would expect to respect the whitespace by default